src/ConfigProvider.php

```diff
@@ -58,11 +59,15 @@ public function getDependencies() : array
                 // AuthenticationInterface::class => Basic\BasicAccess::class,
                 // Provide an alias for the UserRepository adapter based on your application needs.
                 // UserRepositoryInterface::class => UserRepository\Htpasswd::class
+                UserInterface::class => DefaultUser::class
```

Comment:
Trailing comma after the last element in the array, here and in many other places.